reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -17,6 +17,255 @@ def __init__(self):
         self.ff_fn = self.data_dir / "forcefield.xml"
 
 
+@Project.label
+def is_cassandra(job):
+    return job.sp.simulation_engine == "cassandra"
+
+
+@Project.operation
+@Project.post(cassandra_complete)
+@directives(omp_num_threads=4)
+def run_cassandra(job):
+    """"""
+
+    import foyer
+    import mbuild as mb
+    import mosdef_cassandra as mc
+    import unyt as u
+    from mbuild.formats.xyz import read_xyz
+
+    from reproducibility_project.src.molecules.methane_ua import MethaneUA
+    from reproducibility_project.src.molecules.pentane_ua import PentaneUA
+    from reproducibility_project.src.molecules.system_builder import (
+        construct_system,
+    )
+    from reproducibility_project.src.utils.forcefields import load_ff
+
+    compound_dict = {
+        "methaneUA": MethaneUA(),
+        "pentaneUA": PentaneUA(),
+        "benzeneUA": BenzeneUA(),
+        "waterSPC/E": mb.lib.molecules.water.WaterSPC(),
+        "ethanolAA": mb.load("CCO", smiles=True),
+    }
+    molecule = job.sp.molecule
+
+    compound = compound_dict[molecule]
+
+    ensemble = job.sp.production_ensemble
+
+    cass_ensembles = {
+        "NPT": "npt",
+        "GEMC-NVT": "gemc",
+    }
+
+    Nliq = job.sp.N_liquid
+    Nvap = job.sp.N_vap
+    N = Nliq + Nvap
+
+    filled_boxes = construct_system(job.sp)
+
+    liqbox_filled = filled_boxes[0]
+    if ensemble == "GEMC-NVT":
+        vapbox_filled = filled_boxes[1]
+
+    ffname = job.sp.forcefield_name
+    ff = load_ff(ffname)
+    structure = ff.apply(compound)
+
+    T = job.sp.temperature * u.K
+    P = job.sp.pressure * u.kPa
+
+    species_list = [structure]
+    cutoff = job.sp.r_cut * u.nm
+
+    seedslist = [
+        [7860904, 98601355],
+        [955793508, 417823039],
+        [254420642, 2130720464],
+        [58120272, 1465850411],
+        [757616664, 1940492980],
+        [966844679, 1326087693],
+        [1992175335, 1840317929],
+        [650725500, 646331893],
+        [1204247127, 1521385831],
+        [2000946981, 1969870819],
+        [1488434295, 648017520],
+        [1128424221, 1140005446],
+        [58870203, 2133009902],
+        [2024564019, 2014788324],
+        [133927152, 2052536489],
+        [23375750, 1951798462],
+    ]
+
+    seeds = seedslist[job.sp.replica]
+
+    cbmc_n_ins = 12
+    cbmc_n_dihed = 50
+
+    prop_freq = 10
+    coord_freq = 10
+    if ff_name == "oplsaa":
+        comb_rule = "geometric"
+    else:
+        comb_rule = "lb"
+
```

Comment:
You can now access this programmatically if you want.

```suggestion
if ff.combining_rule == "geometric":
    comb_rule = "geometric"
else:
    comb_rule = "lb"
```